### PR TITLE
Update to use PHPUnit 10 syntax

### PR DESCRIPTION
**Pull request by samsonasik**
|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

Updated to latest PHPUnit 10 syntax version based on defined in composer.json

### CODE REVIEW

**Comment by samsonasik** [comment]
Ready for review 👍

File: test/Request/SerializerTest.php [hunk 1]
```diff
@@ -172,11 +173,11 @@ public static function absoluteForms(): array
 
     // @codingStandardsIgnoreStart if we split these line, phpcs can't associate parameter name and docblock anymore (phpcs limitation)
     /**
-     * @dataProvider absoluteForms
      * @param non-empty-string $line
      * @param non-empty-string $requestTarget
      * @param array{getScheme?: non-empty-string, getUserInfo?: non-empty-string, getHost?: non-empty-string, getPort?: positive-int, getPath?: non-empty-string, getQuery?: non-empty-string} $expectations
      */
+    #[\PHPUnit\Framework\Attributes\DataProvider('absoluteForms')]
```
Comment (gsteel) [inline test/Request/SerializerTest.php]: Rogue FQCN here

**Review by gsteel — CHANGES_REQUESTED** [review]
Thanks @samsonasik - Just one cosmetic change 👍

File: test/Request/SerializerTest.php (hunk 1 above)
Comment (samsonasik) [reply test/Request/SerializerTest.php]: updated from use statement 👍 https://github.com/laminas/laminas-diactoros/pull/201/commits/9c809598be4dd7bf7db4390aa14777d9609dcb55

**Review by gsteel — APPROVED** [review]

**Pull request merged**